tflite2onnx/op/reshape.py

```diff
@@ -72,13 +73,45 @@ def parse(self):
 
         self.setParsed()
 
+    def fakeTranspose(self):
+        # If the input (or output) tensor of Reshape
+        # holds a special data layout semantic,
+        # we need to insert Transpose before (or after) the Reshape operator
+        # https://github.com/jackwish/tflite2onnx/issues/28
+
+        # assert(self.status.initialized)
+        to_transpose = self.inputs[0]
+
+        transposed_name = 'TFLITE2ONNX_Transposed_%s' % to_transpose.name
+        transposed = self.TFactory.getWithRef(to_transpose, transposed_name, True)
+        transposed.shape = [to_transpose.shape[p] for p in [0, 2, 3, 1]]
```

Comment:
The shape should be parsed from the `to_transpose.layout` and `to_transpose.shape` rather than assume it's an NHWC. You can get a hint from the [_layout transform_ utils](https://github.com/jackwish/tflite2onnx/blob/master/tflite2onnx/layout.py#L11). I think it should be ready to be called.